### ISSUE TITLE
[feature] : navbar hamburger icon

### ISSUE THREAD

**Issue by Hardik2972** [Anandsg/Hungry-hero#274]
## when we open this website on small screen navbar content not  converted into a hamburger icon which looks messy 
![Screenshot 2023-10-23 195648](https://github.com/Anandsg/Hungry-hero/assets/115338787/d5d38e0d-72ce-4268-9815-95f3ddc466ae)

## I want to add the hamburger icon 
can you please assign it to me

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Anandsg** [comment]
@Hardik2972 We used to have a feature on smaller devices where the navigation bar turned into a hamburger menu. But, to be honest, it didn't look very good, so we decided to keep it the way it is now.

**Comment by Anandsg** [comment]
@JeevaRamanathan Yes we still have that issue would you like to work on it? if so, please create a new issue i will assign that your bucket
